app/experimenter/nimbus-ui/src/components/MonitoringLink/index.tsx

```diff
@@ -11,7 +11,9 @@ export const MonitoringLink: React.FC<Pick<
   "monitoringDashboardUrl"
 >> = ({ monitoringDashboardUrl }) => (
   <>
-    <h3 className="h5 mb-3">Monitoring</h3>
+    <h3 className="h5 mb-3" id="monitoring">
+      Monitoring
+    </h3>
```

Comment:
While you're in this file, this could be named `LinkMonitoring` to match our other naming conventions 🤔